stable/spring-cloud-data-flow/templates/server-rbac.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if .Values.rbac.create -}}
+kind: Role
+apiVersion: rbac.authorization.k8s.io/{{ required "A valid .Values.rbac.apiVersion entry required!" .Values.rbac.apiVersion }}
```

Comment:
Use `v1` instead of making this configurable.

Reply:
updated